libnavigation-core/api/current.txt

```diff
@@ -198,6 +198,13 @@ package com.mapbox.navigation.core {
 
 }
 
+package com.mapbox.navigation.core.accounts {
```

Comment:
@VysotskiVadim @dzinad any idea why this entry was added? Nothing related to this was changed in code.

Reply:
Just metalava living its own life.